pom.xml

```diff
@@ -45,22 +48,140 @@
             <distribution>repo</distribution>
         </license>
     </licenses>
+    <repositories>
+        <repository>
+            <id>sonatype-staging</id>
```

Comment:
Is this one needed?

Reply:
For now, yes, it seems to be necessary to pull in the EE 10 artifacts. Once things settle down in that area, this can be removed.

Reply:
Got it, makes sense.